.github/CODEOWNERS

```diff
@@ -2,15 +2,31 @@
 # Each line is a file pattern followed by one or more owners.
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
+# default owners, overridden by package specific owners below
+* @asaj @cmcewen @timmoreton
+
 # directory and file-level owners. Feel free to add to this!
 /packages/analytics/ @nambrot @timmoreton
+/packages/attestation-service/ @nambrot
+/packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
-/packages/cli/ @mcortesi @asaj
+/packages/celotool/ @asaj
+/packages/cli/ @mcortesi
+/packages/contractkit/ @mcortesi
+/packages/dappkit/ @nambrot
```

Comment:
I'd say at this point I'm so far removed that I think it makes sense for someone on the mobile/sdk side to take this?

Reply:
Any suggestions?